docs/release-notes.rst

```diff
@@ -0,0 +1,205 @@
+IPDK Kubernetes Infrastructure Offload Release Notes
+#############################################################
+
+IPDK 24.01
+************
+
+What's new in this Release
+===========================
+
+- Service Load Balancing: Addition of service type load-balancer feature.
```

Comment:
Replace with - Service Load Balancing: Support for K8s Service of type ClusterIP.